internal/apiserver/route_healcth_check.go

```diff
@@ -0,0 +1,42 @@
+// Copyright © 2022 Kaleido, Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package apiserver
+
+import (
+	"net/http"
+
+	"github.com/hyperledger/firefly-common/pkg/ffapi"
+	"github.com/hyperledger/firefly/internal/coremsgs"
+	"github.com/hyperledger/firefly/pkg/core"
+)
+
+var healthCheck = &ffapi.Route{
+	Name:   "healthCheck",
+	Path:   "health",
+	Method: http.MethodGet,
+	PathParams: []*ffapi.PathParam{},
+	QueryParams:     nil,
+	Description:     coremsgs.APIEndpointsGetNamespace,
+	JSONInputValue:  nil,
+	JSONOutputValue: func() interface{} { return &core.Namespace{} },
```

Comment:
Type doesn't match API.
You can create one similar to https://github.com/hyperledger/firefly-transaction-manager/blob/main/pkg/fftm/route_get_status_live.go#L55